CMakeLists.txt

```diff
@@ -59,21 +59,36 @@ set( CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE )
 set( CMAKE_CUDA_COMPILER  "/usr/local/cuda/bin/nvcc" )
 include( CheckCXXCompilerFlag )
 
-if ( ${CMAKE_SYSTEM_NAME} MATCHES "Darwin" )
-    message( STATUS "============== OSX Environment ================" )
-    set( MACOS TRUE )
-    # Adjust accordingly if libomp is not install here.
-    set( CMAKE_CXX_COMPILER "/usr/local/opt/llvm/bin/clang++" )
-    set( CMAKE_CXX_FLAGS "-I/usr/local/opt/libomp/include" )
-    check_cxx_compiler_flag( -std=c++11 HAS_STD_CPP11_FLAG )
-    if ( HAS_STD_CPP11_FLAG )
-        add_compile_options( -std=c++11 )
-    endif ()
-elseif ( ${CMAKE_SYSTEM_NAME} MATCHES "Linux" )
-    message( STATUS "============== Linux Environment ===============" )
-    set( LINUX TRUE )
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
please keep the format style, 
keep one space after "if";
keep one space after "(" and before ")"

Reply:
Done.